### ISSUE TITLE
RealEstate RGB frame results not reproducible

### ISSUE THREAD

**Issue by 1ssb** [ayushtewari/DFM#8]
![frame_depth_0000](https://github.com/ayushtewari/DFM/assets/40661648/fe326eee-0f2f-4baa-bc7f-af37d92acda0) ![frame_0000](https://github.com/ayushtewari/DFM/assets/40661648/d11acfa0-9837-4224-84fa-c72a051efc8c)
The depth and RGB frames above are what I am getting for the realestate samples at nsamples=2.
Is this expected, should I wait for more samples? The out dictionary shows that the videos all have pixel values between 163-255. Is it over-normalising somewhere?

Either way pose computations are incorrect, there is however a caveat as I broke the dataloading because the loading is not working as posted, so I had to change the steps, but that shouldn't change the nature of the trajectory (ideally).

**Comment by rdzhao** [comment]
Did you figure out a way to run the pre-trained model?

**Comment by 1ssb** [comment]
I broke the dataloader and bypassed the loading with small amount of data,
but both the pose computing and rendering is incorrect (could be
disentangled), so there might be other issues.

Solution: Do not delete the render poses -- They are working now.

Update: You may check the code I provided on the other closed issue to process the images.

**Comment by 1ssb** [comment]
Solution: Do not delete the render poses.
